moveit_ros/moveit_servo/test/unit_test_servo_calcs.cpp

```diff
@@ -520,7 +504,33 @@ int main(int argc, char** argv)
   ::testing::InitGoogleTest(&argc, argv);
   rclcpp::init(argc, argv);
 
+  // Set up the shared node
+  TEST_NODE = std::make_shared<rclcpp::Node>("trying_something");
```

Comment:
I like the idea.  You should change the name of the node though.

Reply:
Ack, fixed